pages/common/adb-pm.md

```diff
@@ -0,0 +1,36 @@
+# pm
+
+> Android Package Manager tool. Use via `adb shell pm` to manage packages, permissions, users, features, and system information.
+> More information: <https://developer.android.com/tools/adb>.
+
+- Print the path to the APK of a package:
+
+`adb shell pm path {{package}}`
+
+- Install an app package from a given path:
+
+`adb shell pm install {{/path/to/app.apk}}`
```

Comment:
```suggestion
`adb shell pm install /{{path/to/app.apk}}`
```
If the path must be absolute, leave the leading slash outside of the placeholder.